### PR TITLE
add-details-to-exception, resolves #485

### LINKED ISSUE #485: Expose DETAIL-message in PGSQLIntegrityConstraintViolationException
Hi, the following in log:
```
ERROR:  duplicate key value violates unique constraint "project_number_key"
DETAIL:  Key (project_number)=(216) already exists.
```
The `DETAIL`-message is not available in `PGSQLIntegrityConstraintViolationException`, it would be great if it would.
Any chance this DETAIL-message could be included?

### PR DESCRIPTION
**Pull request by andreak**
Fixes #485

### CODE REVIEW

File: driver/src/main/java/com/impossibl/postgres/api/jdbc/PGSQLExceptionInfo.java [hunk 1]
```diff
@@ -55,6 +55,10 @@
 
   String getConstraint();
 
-  void setConstraint(String constraint);
```
Comment (kdubb) [inline driver/src/main/java/com/impossibl/postgres/api/jdbc/PGSQLExceptionInfo.java:58]: Unnecessarily renamed?

File: driver/src/test/java/com/impossibl/postgres/jdbc/ServerErrorTest.java [hunk 2]
```diff
@@ -105,6 +105,7 @@ public void testPrimaryKey() throws Exception {
       assertEquals("public", sqle.getSchema());
       assertEquals("testerr", sqle.getTable());
       assertEquals("testerr_pk", sqle.getConstraint());
+      assertEquals("Key (id)=(1) already exists.", sqle.getDetail());
```
Comment (kdubb) [inline driver/src/test/java/com/impossibl/postgres/jdbc/ServerErrorTest.java:108]: Is this going to be too specific to a server version?

File: driver/src/test/java/com/impossibl/postgres/jdbc/ServerErrorTest.java [hunk 3]
```diff
@@ -131,6 +132,7 @@ public void testColumn() throws Exception {
       assertEquals("val", sqle.getColumn());
       assertNull(sqle.getDatatype());
       assertNull(sqle.getConstraint());
+      assertEquals("Failing row contains (1, null).", sqle.getDetail());
```
Comment (kdubb) [inline driver/src/test/java/com/impossibl/postgres/jdbc/ServerErrorTest.java:135]: Same as above...

**Review by kdubb — CHANGES_REQUESTED** [review]

File: driver/src/main/java/com/impossibl/postgres/api/jdbc/PGSQLExceptionInfo.java (hunk 1 above)
Comment (andreak) [reply driver/src/main/java/com/impossibl/postgres/api/jdbc/PGSQLExceptionInfo.java:58]: Hm, that was surely not intended, fixed now.

File: driver/src/test/java/com/impossibl/postgres/jdbc/ServerErrorTest.java (hunk 2 above)
Comment (andreak) [reply driver/src/test/java/com/impossibl/postgres/jdbc/ServerErrorTest.java:108]: Maybe, don't know. Any ideas?

**Review by kdubb — APPROVED** [review]

**Pull request merged**